src/iop/borders.c

```diff
@@ -1011,6 +1011,7 @@ void gui_init(struct dt_iop_module_t *self)
   gtk_box_pack_start(GTK_BOX(box), GTK_WIDGET(g->colorpick), FALSE, TRUE, 0);
   g->border_picker = dt_color_picker_new(self, DT_COLOR_PICKER_POINT, box);
   gtk_widget_set_tooltip_text(GTK_WIDGET(g->border_picker), _("pick border color from image"));
+  dt_action_define_iop(self, NULL, N_("pick border color"), g->border_picker, &dt_action_def_toggle);
```

Comment:
Why not put them inside a "pickers" group as done for negadoctor below?

Reply:
I felt that 3 items was the critical mass to make it worth it to force the user to open a subfolder. Can change of course if you like.

Reply:
Looks cleaner to me to group them as soon as we have more than one. Please, yes I'm for the change. TIA.

Reply:
Done. Added a few more (in blending and curves). I think it's complete now.

And noticed that the "pickers" group in rgb levels never was marked for translation. Fixed that too (although of course it would anyway pick up the translations now marked elsewhere).